app/javascript/src/Types/SwaggerTypes.jsx

```diff
@@ -0,0 +1,50 @@
+export type Primitive = string | number | boolean
+
+export type AccountData = {
+ [string]: Array<{name: string, value: string}>
```

Comment:
```suggestion
 [string]: Array<{ name: string, value: string }>
```
Like in https://github.com/3scale/porta/pull/1991/files#diff-186a85894575b636b4b537b9c7472c57R13